compiler/runtime/Runtime.hpp

```diff
@@ -505,7 +505,8 @@ typedef struct TR_AOTMethodHeader {
 #define TR_AOTMethodHeader_NeedsRecursiveMethodTrampolineReservation 0x00000001
 #define TR_AOTMethodHeader_IsNotCapableOfMethodEnterTracing          0x00000002
 #define TR_AOTMethodHeader_IsNotCapableOfMethodExitTracing           0x00000004
-
+#define TR_AOTMethodHeader_UsesEnableStringCompressionFolding        0x00000008
+#define TR_AOTMethodHeader_StringCompressionEnabled                  0x00000010
 
```

Comment:
New plan: scrap this change altogether and have another one just to split Runtime.hpp into two parts. The J9 specific part will live into a new file called trj9/runtime/Runtime.hpp which will include "compiler/runtime/Runtime.hpp". @0xdaryl  how does that sound?